components/datetime/src/provider/mod.rs

```diff
@@ -28,6 +29,10 @@ pub mod key {
     pub const GREGORY_DATE_PATTERNS_V1: ResourceKey =
         resource_key!(DateTime, "gregory_patterns", 1);
 
+    /// A [`ResourceKey`] to [`gregory::DateSkeletonPatternsV1`].
+    pub const GREGORY_DATE_SKELETON_PATTERNS_V1: ResourceKey =
+        resource_key!(DateTime, "gregory_skltns", 1);
```

Comment:
Suggestion: I don't think abbreviating "skltns" gives many advantages, and makes the output harder to parse and understand for humans. I would suggest on being more verbose here on the name, and spell out "skeletons".

Reply:
It gives the advantage of compiling. Filed as #1148.